### ISSUE TITLE
Make a convertion 

### ISSUE THREAD

**Issue by 9acca9** [janeczku/calibre-web#2818]
I have a library of over 80,000 epub books. I've noticed some don't ship through amazon (amazon rejects them without saying what the problem is). But I noticed that if those same epubs I transform them to .. epub like for example in a page like this:
https://ebook.online-convert.com/en/convert-to-epub
amazon doesn't reject it (I guess it would be the same from the Calibre app).

I don't know how many of the 80,000 have this problem, because I thought they were few, but every once in a while one appears and it's cumbersome because there are several users who access my library, and I have to pay attention to them to fix the books, etc.

So it will be possible to convert to some format? at least with the caliber application system before sending? to do this conversion process if the option existed in configuration (for example).
In summary it is: Will there be the possibility of adding the conversion option to Calibre-Web?
Thanks.

**Comment by OzzieIsaacs** [comment]
The conversion done using calibre itself. Follow the instructions in the wiki: https://github.com/janeczku/calibre-web/wiki/Configuration#path-to-calibre-e-book-converter
Afterwards in the edit books section you can convert the problematic books to e.g. mobi, delete the ePub file and then convert the mobi back to ePub

**Comment by 9acca9** [comment]
oh, excelent! Thanks!
